### PR TITLE
feat: update proto and test code due to panacea-core upgrade

### PR DESCRIPTION
**Pull request by audtlr24**
### change proto file & test code with panacea-core related this PR (https://github.com/medibloc/panacea-core/pull/468)
- error code is changed when tx was not found (`INVALID_ARGUMENT` -> `NOT_FOUND`)
- event is changed when send coin tx

### CODE REVIEW

**Comment by 0xHansLee** [comment]
It would be good to remove all related to `x/token` because the module will be removed in `v2.0.5`.

**Comment by gyuguen** [comment]
Is this status a `Draft` because it's not being deployed at this time?

File: src/test/java/org/medibloc/panacea/GrpcAolTest.java [hunk 1]
```diff
@@ -156,7 +157,7 @@ private TxResponse pollTxResponse(String txHash, int maxTries, int sleepMs) thro
             } catch (StatusRuntimeException e) {
                 // if tx was not found (if tx isn't included in the block yet)
                 // Misc: Yeah. I know that the status code INVALID_ARGUMENT isn't proper, but it's the way how Cosmos was implemented.
```
Comment (youngjoon-lee) [inline src/test/java/org/medibloc/panacea/GrpcAolTest.java:159]: you can delete this `Misc` comment.

File: proto_update_dep.sh [hunk 2]
```diff
@@ -13,8 +13,8 @@ PROTO_PATH="${SCRIPT_DIR}/src/main/resources/proto"
 ##############################################################
 # Download proto files from the 'panacea-core' git repo
 ##############################################################
-PANACEA_CORE_VER_SHORT="2.0.1"
-PANACEA_CORE_VER="v${PANACEA_CORE_VER_SHORT}"
+PANACEA_CORE_VER_SHORT="ft-na-v2.0.5-cosmos-sdk-v0.45.9-dragonberry"
+PANACEA_CORE_VER="ft/na/v2.0.5-cosmos-sdk-v0.45.9-dragonberry"
```
Comment (youngjoon-lee) [inline proto_update_dep.sh]: Now, let's use v2.0.5.

File: src/test/java/org/medibloc/panacea/GrpcAolTest.java (hunk 1 above)
Comment (youngjoon-lee) [inline src/test/java/org/medibloc/panacea/GrpcAolTest.java]: you can delete this `Misc` comment.

**Review by youngjoon-lee — APPROVED** [review]
lgtm. pls make this PR official.

**Review by inchori — APPROVED** [review]
lgtm

**Review by gyuguen — APPROVED** [review]
LGTM

**Pull request merged**